test/tests/fvkernels/mms/skewness-correction/two_term_extrapol/tests

```diff
@@ -0,0 +1,26 @@
+[Tests]
+  design = 'fv_design.md'
+  issues = '#16239'
+  [average]
+    type = PythonUnitTest
+    input = 'test.py'
+    test_case = TestAverageStencil
```

Comment:
what is this for?

Reply:
To show that the simple average stencil is first order.

Reply:
Removed this test since other tests already cover the average stencil on unstructured meshes.